estate/models/estate.py

```diff
@@ -0,0 +1,32 @@
+from odoo import models, fields
+
+
+class EstateModel(models.Model):
+    _name = "estate.property"
```

Comment:
In python, use single quote for technical terms and double quotes for "human" texts. (In JS we always use double quotes so that a linter can check it)